instabot/api/prepare.py

```diff
@@ -23,23 +24,25 @@ def check_secret():
                     login = f.readline().strip()
                     password = f.readline().strip()
                     if len(login) < 4 or len(password) < 6:
-                        print("Data in 'secret.txt' file is invalid. "
+                        
+                        logging.warning("Data in 'secret.txt' file is invalid. "
```

Comment:
Yeah.  Good news is that' the prompts are going to the log file.  Bad news is that they're going to the log file! :dagger: